app/src/main/res/values/strings.xml

```diff
@@ -650,6 +650,7 @@
     <!-- /Description editing tutorial -->
 
     <!-- Suggested edits -->
+    <string name="suggested_edits_activity_title">Suggested edits</string>
```

Comment:
Couldn't we just change the text of the existing string resource instead of creating a new one?

Reply:
Done!